app/presenters/caution_result_presenter.rb

```diff
@@ -3,6 +3,19 @@ def to_partial_path
     'results/caution'
   end
 
+  def variant
```

Comment:
No need to override this method. You only want the value of the `expiry_date` returned by the `CautionCalculator` to be `:simple` to accomplish the same.
Also, the `#caution_type` method to the bottom can be removed as will not be necessary.